### DISCUSSION TITLE
GSOC 2019 Page Builder

### DISCUSSION THREAD

**Discussion by barsan-md** [joomla/joomla-cms#34471] [General] [answered]
Searching through joomla 4 documentation I came across a project for a Page Builder developed during the GSOC 2019.
https://docs.joomla.org/J4.x:Page_Builder
https://github.com/joomla-projects/gsoc19_page-builder
I'm wondering what is the state of that project and if are there any plans or interest to further develop it and integrate it into the core.

**Comment by bembelimen** [comment, marked as answer]
Sadly when we wanted to start the 2nd part of this project we were not selected in GSoC, so currently it's on Pause.... if someone is interested in restarting it again, we have a lot of ideas and concepts already prepared. But for now there is no progress to expect, especially due to the J! 4.0 release coming soon.

**Reply by barsan-md** [reply]
What is the nature of it? I mean is that a set of extensions like a component, a plugin, or a template. Or is it rightly coupled with the core and cannot be installed separately?
Thanks for the information.

**Reply by bembelimen** [reply]
It's a template which allows the user to define the structure by a drag&drop editor.
